dev-tools/omicron-dev/tests/test-omicron-dev.rs

```diff
@@ -139,7 +139,7 @@ async fn test_run_all() {
     let cmd_path = path_to_omicron_dev();
 
     let cmdstr = format!(
-        "( set -o monitor; {} run-all --nexus-listen-port 0 && true )",
+        "( set -o monitor; {} run-all --nexus-listen-port 0 --nexus-config ../../nexus/examples/config.toml && true )",
```

Comment:
It seems like the command itself has a default built into it so I feel like you shouldn't need this here?  Or is the problem that the relative paths are wrong (e.g., they're different between this context and the common `cargo xtask omicron-dev run-all` context)?  In that case should we figure out the right path based on CARGO_HOME?

This seems like not a big deal, I guess.